Libraries/Utilities/StringExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Orchard.Environment.Extensions;
+
+namespace System
+{
+    /// <summary>
+    /// This class provides useful methods for string usage and manipulation, usually safe to use with empty and <see
+    /// langword="null"/> strings.
+    /// </summary>
+    [OrchardFeature("Piedone.HelpfulLibraries.Utilities")]

```

Comment:
```suggestion

```
OrchardFeature attribute is useless on a static class.

Reply:
That makes sense 😀

Reply:
But still: `AsyncHelper`, `ContainerBuilderExtensions`, and probably other `static` classes in the _Utilities_ folder **do** carry that attribute. So - what gives?

Reply:
`OrchardFeature` only applies/matters to stuff that descends from `IDependency`, so it's just an oversight.

Reply:
I'll remove it from all the static classes in Utilities then, OK?

Reply:
Hm, you were fast! What now?

Reply:
> I'll remove it from all the static classes in Utilities then, OK?

Allrighty!